x/evmutil/types/msg.go

```diff
@@ -12,17 +12,26 @@ import (
 // ensure Msg interface compliance at compile time
 var (
 	_ sdk.Msg            = &MsgConvertCoinToERC20{}
-	_ sdk.Msg            = &MsgConvertERC20ToCoin{}
 	_ legacytx.LegacyMsg = &MsgConvertCoinToERC20{}
+	_ sdk.Msg            = &MsgConvertERC20ToCoin{}
 	_ legacytx.LegacyMsg = &MsgConvertERC20ToCoin{}
+
+	_ sdk.Msg            = &MsgConvertNativeCoinToERC20{}
```

Comment:
prefer `Cosmos` vs Native as conceptually Kava is a co-chain that "natively" supports evm and cosmos blockchains but in the future might also allow for converting things like BTC more directly

Reply:
i support the name change. it's big a big enough change set to move references from "Native Coin" to "Cosmos Coin" that i want to make it in a follow up pr that changes all the unreleased types (like the allowed_native_denoms param), not just the one for this message.

Reply:
https://github.com/Kava-Labs/kava/pull/1596